allennlp/commands/elmo.py

```diff
@@ -273,29 +275,28 @@ def embed_file(self,
         assert output_format in ["all", "top", "average"]
 
         # Tokenizes the sentences.
-        sentences = [line.strip() for line in input_file if line.strip()]
+        sentences = [line.strip() for line in input_file]
+        if "" in sentences:
```

Comment:
I'm pretty sure `in` on a `list` is linear time --- if we're doing this anyway, it might be helpful to just iterate through the lines explicitly and point out the specific line a user might want to look at?